packages/@webex/plugin-meetings/src/meetings/collection.ts

```diff
@@ -60,4 +60,17 @@ export default class MeetingCollection extends Collection {
 
     return null;
   }
+
+  /**
+   * Gets the meeting that has a webrtc media connection
+   * NOTE: this function assumes there is no more than 1 such meeting
+   *
+   * @returns {Meeting} first meeting found, else undefined
+   * @public
+   * @memberof MeetingCollection
+   */
+  public getActiveWebrtcMeeting() {
+    // @ts-ignore
+    return find(this.meetings, (meeting) => meeting.mediaProperties.webrtcMediaConnection);
```

Comment:
question: why do not you use `filter` it would give you all the connections

Reply:
right now I think we only support joining 1 meeting at a time (for sure in the web app, not sure about SDK, probably as well, at least the SDK sample app doesn't support more). If I used a filter, it would return an array and every time I would have to add `[0]` at the end in my console command. The API at the meeting level is private (and MeetingCollection is private) so we can easily change it in the future if we ever support joining more meetings in parallel